torchani/ase.py

```diff
@@ -73,9 +73,11 @@ def calculate(self, atoms=None, properties=['energy'],
 
         if 'forces' in properties:
             forces = -torch.autograd.grad(energy.squeeze(), coordinates, retain_graph='stress' in properties)[0]
+            forces *= ase.units.Hartree
```

Comment:
energy is already in eV, and forces is computed from energy using autograd, so the force should also be in eV? Same thing applies to stress

Reply:
Ahhhhhh
that makes a lot of sense, thanks
nothing was making sense before, all the frequencies were wrong
disregard this PR then